src/client.rs

```diff
@@ -218,6 +230,9 @@ pub enum ClientFlowEvent {
     StatusReceived {
         status: Status<'static>,
     },
+    ContinuationReceived {
+        continuation: CommandContinuationRequest<'static>,
+    },
```

Comment:
IMO we should start documenting stuff. And this variant is worth to be documented because it's a little bit complicated: "Continuation that was not consumed by `ClientFlow`..."